Bottom: 68ab956fe27c2cfef62665fbf7b5845d8ce483b1
Top:    fe1d01c353992b61fcdc1a0ab729805c12e9592f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 16:22:11 +0000

*.l: Add #define YY_NO_INPUT to avoid defining unused "input()" function

Silences compiler warnings


---

diff --git a/src/parse_l.l b/src/parse_l.l
index b1b67a8..c9d96c8 100644
--- a/src/parse_l.l
+++ b/src/parse_l.l
@@ -62,6 +62,8 @@ RCSID("$Id$");
 #include "parse_l.h"
 #include "parse_y.h"
 
+#define YY_NO_INPUT
+
 /* ---------------------------------------------------------------------------
  * some shared parser identifiers
  */
